Dockerfile

```diff
@@ -232,52 +251,57 @@ RUN ldconfig
 # This should only install packages that are specifically needed for the dev environment and nothing else
 # Do you really need to add another package here? Can it be done in a different build stage?
 RUN apt-get update && apt-get install -y --no-install-recommends \
-	apparmor \
-	aufs-tools \
-	bash-completion \
-	btrfs-tools \
-	iptables \
-	jq \
-	libcap2-bin \
-	libdevmapper-dev \
-	libudev-dev \
-	libsystemd-dev \
-	binutils-mingw-w64 \
-	g++-mingw-w64-x86-64 \
-	net-tools \
-	pigz \
-	python3-pip \
-	python3-setuptools \
-	python3-wheel \
-	thin-provisioning-tools \
-	vim \
-	vim-common \
-	xfsprogs \
-	zip \
-	bzip2 \
-	xz-utils \
-	libprotobuf-c1 \
-	libnet1 \
-	libnl-3-200 \
-	&& rm -rf /var/lib/apt/lists/*
+        apparmor \
+        aufs-tools \
+        bash-completion \
+        binutils-mingw-w64 \
+        libbtrfs-dev \
+        bzip2 \
+        g++-mingw-w64-x86-64 \
+        iptables \
+        jq \
+        libcap2-bin \
+        libdevmapper-dev \
+        libnet1 \
+        libnl-3-200 \
+        libprotobuf-c1 \
+        libsystemd-dev \
+        libudev-dev \
+        net-tools \
+        pigz \
+        python3-pip \
+        python3-setuptools \
+        python3-wheel \
+        thin-provisioning-tools \
+        vim \
+        vim-common \
+        xfsprogs \
+        xz-utils \
+        zip \
+    && rm -rf /var/lib/apt/lists/*
+
+# Switch to use iptables instead of nftables (to match the host machine)
```

Comment:
It would be nice if we could somehow detect this and use the appropriate version accordingly instead of just assuming the host uses `iptables` :disappointed:

(maybe a `TODO` for if/when we support nftables?)

Reply:
Yeah; this one was mostly taking from what's on master, but I agree that this is pretty much hard coded to the machines we currently have running in our CI

Perhaps you could open a quick PR on master to add the TODO (and a corresponding tracking issue?)

Reply:
Oh doh, forgot this was a backport, sorry!  Will do! :+1: